charts/dapr/README.md

```diff
@@ -63,7 +63,7 @@ The Helm chart has the follow configuration options that can be supplied:
 | Parameter                                 | Description                                                             | Default                 |
 |-------------------------------------------|-------------------------------------------------------------------------|-------------------------|
 | `global.registry`                         | Global Dapr docker image registry                                       | `docker.io/daprio`      |
-| `global.tag`                              | Global Dapr docker image version tag                                    | `0.11.0`                |
```

Comment:
We are not publishing RC's as the latest version to be installed *by default*. Users might ask for it as a parameter.

Reply:
@artursouza I see, but this was confusing.  The values.yaml file currently sets the default to "1.0.0-rc.1"  should that be rolled back?

Reply:
You're right, that is confusing. We will rollback values.yaml to 0.11.3.